src/string.rs

```diff
@@ -606,12 +603,51 @@ impl String {
     let buffer_len = buffer.len().try_into().ok()?;
     unsafe {
       scope.cast_local(|sd| {
-        v8__String__NewExternalOneByte(
-          sd.get_isolate_ptr(),
+        let ptr = crate::support::cpp_new::<
+          crate::binding::RUST_ExternalOneByteString,
+        >();
+        crate::binding::RUST_ExternalOneByteString_RUST_ExternalOneByteString(
+          ptr,
```

Comment:
Kind of a pain. Wanted the equivalent of `new RUST_ExternalOneByteString(..)`, which meant I had to allocate and then call the raw constructor binding that takes a pointer to construct into